### PR TITLE
Rayenbou ticket bundle

### PR DESCRIPTION
**Pull request by RayenBou**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | https://packagist.org/packages/rayenbou/ticket-bundle

### CODE REVIEW

File: rayenbou/ticket-bundle/1.0/config/packages/rayenbou_ticket.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+rayen_bou_ticket:
+    authentication:
+        url: "%env(RAYENBOU_TICKET_URL)%"
+        username: "%env(RAYENBOU_TICKET_USERNAME)%"
+        password: "%env(RAYENBOU_TICKET_PASSWORD)%"
+    settings:
```
Comment (Neirda24) [inline rayenbou/ticket-bundle/1.0/config/packages/rayenbou_ticket.yaml:6]: empty ? or does the next should be indented ?

Comment (RayenBou) [reply rayenbou/ticket-bundle/1.0/config/packages/rayenbou_ticket.yaml:6]: I'm currently building a bigger side of this bundle that may need other settings , so i've indented to put more key in future

**Comment by RayenBou** [comment]
first of all i apologize if the many close/reopen  put a little mess. I've tried to test several way to pass those test by removing requirement. But i'm in front of a wall right now:

since i've built my bundle under Symfony 7 , using the AbstractBundle class and the configure/loadExtension directly in my main class , i have no way to make it working under symfony 6.4/php 8.1. But one test require it to work on older version.

thanks in advence

**Comment by Neirda24** [comment]
@RayenBou : maybe just close it and re-open it once you are ready.

**Pull request closed without merging**